File: code_challenge_solution/c#/21 Calculate age in seconds.cs
Here is the C# code to calculate age in seconds:

```csharp
using System;

class Program
{
    static void Main()
    {
        Console.WriteLine("Enter your birth year:");
        int birthYear = int.Parse(Console.ReadLine());

        Console.WriteLine("Enter your birth month:");
        int birthMonth = int.Parse(Console.ReadLine());

        Console.WriteLine("Enter your birth day:");
        int birthDay = int.Parse(Console.ReadLine());

        DateTime birthDate = new DateTime(birthYear, birthMonth, birthDay);
        TimeSpan age = DateTime.Now - birthDate;
        double ageInSeconds = age.TotalSeconds;

        Console.WriteLine("Your age in seconds is: " + ageInSeconds);
    }
}
```

File: code_challenge_solution/c++/21 Calculate age in seconds.cpp
#include <iostream>

int main() {
    int age, seconds;

    std::cout << "Enter your age: ";
    std::cin >> age;

    seconds = age * 365 * 24 * 60 * 60;

    std::cout << "Your age in seconds is: " << seconds << std::endl;

    return 0;
}